lib/atomic_web/live/home_live/components/schedule.ex

```diff
@@ -1,67 +1,141 @@
-                    <.icon name="hero-clock-solid" class="size-4" />
-                    <p class="text-xs font-semibold">
-                      <%= pretty_display_date(entry.start) %>
+                <li class="space-y-3 pt-4">
+                  <div class="flex justify-between">
+                    <p class="text-md font-semibold hover:underline">
+                      <%= entry.title %>
                     </p>
+                    <div class="w-[110px] flex h-6 items-center justify-center space-x-1 rounded-md bg-orange-100 text-orange-500 opacity-70">
+                      <.icon name="hero-calendar-solid" class="size-4" />
+                      <p class="text-xs font-semibold">
+                        <%= pretty_display_date(entry.start) %>
+                      </p>
+                    </div>
                   </div>
                   <p class="text-justify text-sm text-gray-700">
-                    <%= entry.description %>
+                    <%= maybe_slice_string(entry.description, 150) %>
                   </p>
                 </li>
               </.link>
+              <%= if check_enrolled(entry, @current_user) do %>
+                <div class="flex justify-between pt-2">
+                  <.icon name="hero-user-group-solid" class="size-4 font-bold text-green-500" />
+                  <.link navigate={~p"/organizations/#{entry.organization_id}"} class="text-xs text-zinc-400 hover:underline">
+                    <%= entry.organization.name %>
+                  </.link>
+                </div>
+              <% else %>
+                <div class="pt-2 text-right">
+                  <.link navigate={~p"/organizations/#{entry.organization.id}"} class="text-xs text-zinc-400 hover:underline">
+                    <%= entry.organization.name %>
+                  </.link>
+                </div>
+              <% end %>
             <% end %>
           </ul>
         </div>
       </div>
     </div>
     """
   end
+
+  defp check_enrolled(_entry, nil), do: false
+  defp check_enrolled(entry, user), do: Activities.participating?(entry.id, user.id)
+
+  defp show_empty(assigns) when assigns.tab == "user" do
+    ~H"""
+    <%= if @current_user do %>
+      <p>
+        <%= gettext("Nothing to do in the next week.") %>
+      </p>
+      <p>
+        <%= gettext("Try enrolling in some activities.") %>
+      </p>
+      <.button patch={~p"/activities"} color={:white} size={:xs} icon="hero-academic-cap">
+        <%= gettext("Browse activities") %>
+      </.button>
+    <% else %>
+      <p>
+        <%= gettext("You need to be logged in to see your schedule.") %>
+      </p>
+      <.button patch={~p"/users/log_in"} icon="hero-arrow-right-end-on-rectangle-solid" color={:white} icon_position={:right} size={:xs}>
+        <%= gettext("Log In") %>
+      </.button>
+    <% end %>
```

Comment:
Remove "not logged in" state (as mentioned in the comment above) and make the button match other buttons in the section:

![image](https://github.com/user-attachments/assets/b31ac711-3ba5-4ad4-8bfc-1d2200c81b27)


```suggestion
      <p>
        <%= gettext("Nothing to do in the next week.") %>
      </p>
      <p>
        <%= gettext("Try enrolling in some activities.") %>
      </p>
      <.button patch={~p"/activities"} color={:white} size={:md} full_width icon="hero-academic-cap">
        <%= gettext("Browse activities") %>
      </.button>
```